src/Components/SingleCryptid.js

```diff
@@ -1,16 +1,54 @@
 import React, { useEffect, useContext } from 'react'
 import '../Styles/SingleCryptid.scss'
 import { NavigationContext } from '../Context/NavigationContext'
+import {gql, useQuery} from '@apollo/client'
+import bigfoot from '../assets/bigfoot.jpg'
+import { useParams } from 'react-router-dom'
 
-const SingleCryptid = () => {
-  const { setClick } = useContext(NavigationContext)
+const GET_SINGLE_CRYPTID = gql`
```

Comment:
Great job taking the lead on this, I'm going to mess around with this and see what I can do with making the single cryptoids show